designs/tinkerbell-provider-input.md

```diff
@@ -0,0 +1,366 @@
+# Required configuration for EKS Anywhere Tinkerbell provider
+
+## Introduction
+
+The purpose of this document is to explain the different input files required to create an EKS Anywhere Tinkerbell cluster and how to generate them.
+
+### Goals and objectives
+
+As an EKS Anywhere Tinkerbell provider user, I want to:
+
+* Generate the required cluster configuration file with minimal customizations
+* Generate the required hardware files using CSV files that contain hardware information
+
+### Statement of Scope
+
+The scope of this document is mainly for beta release of Tinkerbell provider. Some of the details in the design will change once Tinkerbell is [migrated completely to Kubernetes](https://github.com/tinkerbell/proposals/tree/main/proposals/0026).
```

Comment:
This is good, but could we add a few things:

* Could we specify what are the parts that might change?
* Could we explain why migrating to kubernetes might change this design?
* I think we mentioned previously that the GA design will depend on ongoing conversations/designs. The linked proposal has already been approved. Does that mean we can already design the GA solution even if we can't implement it? Or are there actual unknowns pending on separate discussions?

Reply:
The proposal has been approved but we don't know if all of that will be implemented before GA. I will add a few more details around what we are expecting to change but that may also change later

Reply:
Updated the doc. Let me know if i missed something